server/src/metadata/migration-runner/migration-runner.service.ts

```diff
@@ -116,17 +116,17 @@ export class MigrationRunnerService {
             default: 'public.uuid_generate_v4()',
           },
           {
-            name: 'created_at',
+            name: 'createdAt',
```

Comment:
The query to pg_graphql was wrong because we don't have a mapping for default fields from camelCase to snake_case. To fix that we actually decided to create columns in camelCase to align with the rest of the DB, it's also the standard naming strategy of typeorm. This also means we will need to migrate metadata tables in an upcoming PR